docs/package_firewall/faq.md

```diff
@@ -0,0 +1,56 @@
+# Frequently Asked Questions
+
+## What do I do when a dependency fails analysis?
+
+If a package's installation gets blocked by the firewall, possible malicious
+code execution will have been blocked, so there's no immediate need for action.
+
+Should you determine the policy violation to be a false-positive, you can add an
+exception by navigating to [the firewall overview], selecting the group
+associated with the project, then clicking on the "Allow Package" button in the
+top-right.
+
+[the firewall overview]: https://app.phylum.io/firewall
+
+## Why are some packages violating a policy not being blocked?
+
+The package firewall acts as a proxy between the local package manager and the
+official registry. As a result only packages being retrieved from the official
+registry will be analyzed.
+
+Common scenarios where a package will not be analyzed are the usage of a cache
+between the package firewall and the local client (like Artifactory, or Nexus),
+and the installation from the local registry cache. While the initial
+installation would be analyzed by the package firewall, once cached it will not
+be re-analyzed.
+
+Currently some ecosystems also bypass analysis when a package is installed
+directly from a lockfile. This is why new dependencies should always be
+installed with the package firewall in place, making sure that the lockfile
+contains no packages which violate your policy.
```

Comment:
I contemplated whether I should make this its own section or not, but I think it's beneficial to have all these limitations in the same place and it will also make it trivial to nuke in the future once this restriction is removed.

Reply:
I think it is fine to put it here. We might want to add a little more explanation about how some lockfiles contain direct download URLs and that is why it would bypass the firewall. That might help connect the dots for people.

Reply:
Honestly I'm hoping to keep it minimal since my goal is to fix this in the next couple of weeks anyway.

Reply:
Fine by me!